### PR TITLE
🌐 Add Russian translation for `docs/features.md`

### PR DESCRIPTION
**Pull request by dukkee**
Hey. @TemaSpb, @solomein-sv, @BugLight if you are free, please review :smiley:

### CODE REVIEW

File: docs/ru/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+## FastAPI возможности
+
+**FastAPI** дает вам следующее:
+
+### Использование открытых стандартов
+
+* <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI</strong></a> для создания API, включая объявления <abbr title="также известные как ендпоинты, маршруты">путей</abbr> <abbr title="также известных как HTTP-методы POST, GET, PUT, DELETE">операций</abbr>, параметров, тела запросов, безопасности и т.д.
+* Автоматическое документирование моделей данных с <a href="https://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema</strong></a> (т.к. OpenAPI основан на JSON Schema).
+* Проектирование вокруг этих стандартов после тщательного изучения.
+* Это также позволит автоматически **генерировать код клиента** на многих языках программирования.
```
Comment (solomein-sv) [inline docs/ru/docs/features.md:12]: Автоматическая **генерация клиентского кода** на многих языках программирования.

File: docs/ru/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+Другими словами, в них нет необходимости, просто импортируйте и используйте код, который вам необходим.
+
+Любая интеграция спроектирована таким образом, чтобы быть настолько простой в использовании (с зависимостями), что вы можете создать «плагин» для своего приложения в пару строк кода, используя ту же структуру и синтаксис, что и для ваших *операций с путями*.
+
+### Протестированный
+
+* 100% <abbr title="Количество автоматически протестированного кода">покрытие тестами</abbr>.
+* 100%-<abbr title="Аннотации Python-типов, с которыми ваш редактор и другие инструменты могут предоставить вам лучшую поддержку">покрытая аннотациями типов</abbr> кодовая база.
+* Использован в реальных приложениях.
```
Comment (solomein-sv) [inline docs/ru/docs/features.md:156]: Используется

**Review by solomein-sv — APPROVED** [review]

**Comment by solomein-sv** [comment]
#1362

File: docs/ru/docs/features.md (hunk 1 above)
Comment (dukkee) [reply docs/ru/docs/features.md:12]: @solomein-sv почему? оригинал-то звучит следующим образом `This also allows using automatic client code generation in many languages.`

Comment (solomein-sv) [reply docs/ru/docs/features.md:12]: Потому что дословный перевод не всегда подходит. Потому что перед этим списком идёт "**FastAPI** дает вам следующее:" и предполагается перечисление того, что даёт фастапи. И "Это также позволит ..." подойдёт, если это новый абзац, а не элемент списка. Не по-русски звучит, ИМХО.
Хотя, можно и оставить "вылизывание" перевода на следующий этап :-)

**Review by solomein-sv — APPROVED** [review]

File: docs/ru/docs/features.md (hunk 1 above)
Comment (dukkee) [reply docs/ru/docs/features.md:12]: @solomein-sv у меня больше несогласия с "клиентским кодом", а не "кодом клиента" на самом деле :) по поводу формулировки начала и списка я согласен

Comment (solomein-sv) [reply docs/ru/docs/features.md:12]: Имхо, непринципиально:

Автоматическое создание кода клиента на многих языках программирования.

Пойдет?

Comment (dukkee) [reply docs/ru/docs/features.md]: @solomein-sv исправил

**Review by solomein-sv — APPROVED** [review]

**Comment by h0rn3t** [comment]
I think need to delete russian documentation because of
fascist war against Ukraine, 38 days of war was died more then 150 children and many civilian, many towns was destroyed

**Comment by BugLight** [comment]
> I think need to delete russian documentation because of fascist war against Ukraine, 46 days of war was died more then 250 children and many civilian, many towns was destroyed

Do you really think it is right? There are many russian language speakers outside of Russian Federation. They have nothing to do with that situation.

**Comment by h0rn3t** [comment]
> > I think need to delete russian documentation because of fascist war against Ukraine, 46 days of war was died more then 250 children and many civilian, many towns was destroyed
> 
> Do you really think it is right? There are many russian language speakers outside of Russian Federation. They have nothing to do with that situation.

I am one of those Russian speakers, but I am ashamed to speak and read in the language of fascism

**Comment by TemaSpb** [comment]
> > > I think need to delete russian documentation because of fascist war against Ukraine, 46 days of war was died more then 250 children and many civilian, many towns was destroyed
> > 
> > 
> > Do you really think it is right? There are many russian language speakers outside of Russian Federation. They have nothing to do with that situation.
> 
> I am one of those Russian speakers, but I am ashamed to speak and read in the language of fascism

So don't speak or write on it. If you think people are fascists because they speak a certain language or they live in a certain country, then something is clearly wrong with you.

**Comment by h0rn3t** [comment]
> > > > I think need to delete russian documentation because of fascist war against Ukraine, 46 days of war was died more then 250 children and many civilian, many towns was destroyed
> > > 
> > > 
> > > Do you really think it is right? There are many russian language speakers outside of Russian Federation. They have nothing to do with that situation.
> > 
> > 
> > I am one of those Russian speakers, but I am ashamed to speak and read in the language of fascism
> 
> So don't speak or write on it. If you think people are fascists because they speak a certain language or they live in a certain country, then something is clearly wrong with you.

those soldiers that killing children speak russian and live in russia and come from russia

**Comment by IvanOsyanin** [comment]
> > > > > I think need to delete russian documentation because of fascist war against Ukraine, 46 days of war was died more then 250 children and many civilian, many towns was destroyed
> > > > 
> > > > 
> > > > Do you really think it is right? There are many russian language speakers outside of Russian Federation. They have nothing to do with that situation.
> > > 
> > > 
> > > I am one of those Russian speakers, but I am ashamed to speak and read in the language of fascism
> > 
> > 
> > So don't speak or write on it. If you think people are fascists because they speak a certain language or they live in a certain country, then something is clearly wrong with you.
> 
> those soldiers that killing children speak russian and live in russia and come from russia

Discriminating against people just because of nationality, language or country of origin, or even feeling ashamed about it is a kinda strange, simply because these are things that no one chooses.

**Comment by impocode** [comment]
@dukkee

Привет! Недавно слили перевод этого файла. Думаю, можно закрыть PR, чтоб с толку не сбивало когда смотришь открытые PR'ы для Русского языка.

**Pull request closed without merging**